lib/diagnostic.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (enum).inject { |sum, n| sum + n }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
-  def some_odd(numbers)
-    # Your code goes here
+  def some_odd(numbers)   # one or more
+    numbers.any? { |num| num % 2 != 0 }
```

Comment:
Correct. Numbers actually have a method that gives whether they are odd, `.odd?`, so you could rewrite this as `numbers.any?(&:odd?)`;